unstructured/partition/pdf_image/pdfminer_processing.py

```diff
@@ -102,6 +103,10 @@ def merge_inferred_with_extracted_layout(
     )
     from unstructured_inference.models.detectron2onnx import UnstructuredDetectronONNXModel
 
+    # If the model is a chipper model, we don't want to order the
+    # elements, as they are already ordered
+    order_elements = not hi_res_model_name.startswith("chipper")
```

Comment:
Are there any other future models we'd need to skip ordering for? Key value was the one I had in mind. cc @leah1985 @LaverdeS

Reply:
I only know `chipper` models as of now, but if there are any other future models, we could skip ordering for those models in a separate PR later.

Reply:
👍 - works for me, thanks!